torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -91,15 +97,17 @@
             const int64_t m_end = std::min((mc + Mc_blocks) * M0, M);
             const int64_t m_size = m_end - m_start;
             {%- if use_local_acc %}
-            {{ kernel.define_buffer(acc_buf_name, ["m_end - m_start", "N0"]) }}
+            {%- set buffer_size = ([1] if is_bmm else []) + ["m_end - m_start", "N0"] %}
```

Comment:
Can we pass tensor slices to the GEMM template so that we don't have to complicate it with "batched" semantics here? The "batched" semantics are handled outside of it anyway?